src/main/java/Pages/App/App.java

```diff
@@ -0,0 +1,20 @@
+package Pages.App;
+
+import Pages.Services.InternalServices.DigitalIdRequests.NewId.NewDigitalIdRequestPage;
+import Pages.Services.InternalServices.DigitalIdRequests.NewId.Steper.IdType.IdTypeStepPage;
+import Pages.Services.InternalServices.InternalServicesPage;
+import Pages.Services.ServicesPage;
+
+public class App {
+    public ServicesPage services;
+    public InternalServicesPage internalServices;
+    public NewDigitalIdRequestPage newDigitalIdRequest;
+    public IdTypeStepPage idTypeStep;
+
+    public App() {
+        services = new Services();
+        internalServices = new InternalServices();
+        newDigitalIdRequest = new NewDigitalIdRequest();
+        idTypeStep = new IdTypeStep();
```

Comment:
idTypeStep = new IdTypeStep(); should be created not here but inside of NewDigitalIdRequest
this stepper does not exist outside of creating new digital id

Reply:
same valid for instance of InternalServices, it's a part of new Services() 

later your usage would be smth like 
app.services.internalServices.newDigitalId.idTypeStep ....

so think about it as it's developed and it looks like in your UI. e.g. if you can not deal with internalServices until you go to Services, then it has be inside of Services. The same logic for any other page/component

Reply:
Done.